imap_processing/spice/geometry.py

```diff
@@ -372,6 +372,51 @@ def frame_transform(
     return result
 
 
+def frame_transform_az_el(
+    et: Union[float, npt.NDArray],
+    az_el: npt.NDArray,
+    from_frame: SpiceFrame,
+    to_frame: SpiceFrame,
+    degrees: bool = True,
+) -> npt.NDArray:
+    """
+    Transform azimuth and elevation coordinates between reference frames.
+
+    Parameters
+    ----------
+    et : float or np.ndarray
+        Ephemeris time(s) corresponding to position(s).
+    az_el :  np.ndarray
+        <azimuth, elevation> vector or array of vectors in reference frame `from_frame`.
+        There are several possible shapes for the input az_el and et:
+        1. A single az_el vector may be provided for multiple `et` query times
+        2. A single `et` may be provided for multiple az_el vectors,
+        3. The same number of `et` and az_el vectors may be provided.
+        It is not allowed to have n az_el vectors and m `et`, where n != m.
+    from_frame : SpiceFrame
+        Reference frame of input coordinates.
+    to_frame : SpiceFrame
+        Reference frame of output coordinates.
+    degrees : bool
+        If True, azimuth and elevation input and output will be in degrees.
+
+    Returns
+    -------
+    to_frame_az_el : np.ndarray
+        Azimuth/elevation coordinates in reference frame `to_frame`.
```

Comment:
Describing the shape and order of this output array in a bit more detail would be good - e.g.

```
First axis represents different points from the original az/el mapping
Second axis has length 2, representing (azimuth, elevation) of these
points when projected into to_frame. 
```